### PR TITLE
Feature/11 admin

### LINKED ISSUE #11: ODRC: Toevoegen themas
**Description (why)**
uit PSA: Een beheerder zal in het ODRC thema’s willen kunnen toevoegen, raadplegen, wijzigen en (de)activeren. Thema’s moeten in de data-opslag vastgelegd worden en publicaties moeten hieraan gekoppeld kunnen worden. Deze informatie moet via de API’s ontsloten worden.

**Acceptance criteria (what)**

**Specific details (how)**

**Test plan**

Waardelijst: https://identifier.overheid.nl/tooi/set/scw_toplijst

Note: a theme can be a child of another theme, and the JSON structure above doesn't rule out that a theme is a (direct) child of multiple different other themes

Theme properties:

* `identifier`: URI
* `name`: name of the theme
* `subthema's`: array of nested themes, which may all have their own sub themes (!)
* `publicatieStatus`: actief/niet_actief
* `origin`: enum, `waardelijst` or `zelf_toegevoegd`

**Tasks**

- [x] Define data model for `Thema` and register in the admin. Probably best to use django-treebeard for the tree structure (MPTT)
- [x] Expose via API at `/api/v1/themas`
    - List endpoint is paginated
    - List endpoint has search parameter on `name`
    - List endpoint includes nested themes if relevant, for each item in the list
    - Create/update for origin `zelf_toegevoegd`, waardelijst records may not be mutated
    - publicatieStatus can be updated irrespective of `origin`
    - When creating a theme, you must be able to specify it needs to be a child of another theme (and specify before/after sibling)
- [x] Add management command to populate themes from waardelijst
    - Must produce a fixture, with natural keys
    - Fixture needs to be baked in docker image and automatically loaded on application start (post-migrate signal)

### PR DESCRIPTION
**Pull request by bart-maykin**
Partially Fixes #11 

depends on #46 

**Changes**

Added treebeard
Added theme admin page
Added tests for theme admin

### CODE REVIEW

File: src/woo_publications/templates/admin/tree_change_list.html
Comment (bart-maykin) [inline src/woo_publications/templates/admin/tree_change_list.html:1]: This file is made to alter the tree_change_list.html page within the treebeard library.
This template only gets used currently in conjuction with the Theme's models.
Which should only display the loaded in data from the government, so we want to restrict the user from alter the data.
Hence why I altered this file to remove the js files, which handles requests from the drag and drop feature.

Comment (sergei-maertens) [reply src/woo_publications/templates/admin/tree_change_list.html:1]: can you add this comment to the template file itself too, for the future?

File: src/woo_publications/metadata/tests/test_theme_admin.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+        self.assertEqual(response.status_code, 403)
+
+    def test_theme_admin_can_not_create(
+        self,
+    ):
+        url = reverse("admin:metadata_theme_add")
+
+        response = self.app.get(url, user=self.user, expect_errors=True)
+        self.assertEqual(response.status_code, 403)
```
Comment (sergei-maertens) [inline src/woo_publications/metadata/tests/test_theme_admin.py:94]:
```suggestion
        url = reverse("admin:metadata_theme_add")

        response = self.app.get(url, user=self.user, expect_errors=True)
        
        self.assertEqual(response.status_code, 403)
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
Can you add a django fixture for now (for demo purposes) in `src/woo_publications/fixtures/themes.json` with some of the themes from the waardelijst? 3 or so is enough, so that we can show something in the admin!

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**